src/pages/[platform]/build-a-backend/add-aws-services/rest-api/test-api/index.mdx

```diff
@@ -0,0 +1,144 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+                  
+export const meta = {
+  title: 'Test the REST API',
+  description: 'Learn how you can test the REST API from the terminal, with Amplify Mock, or with the API Gateway console.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+          
+
+## Test the API from the terminal
+
+If Guest users have access to your REST API you can test it from the terminal using Curl.
+
+[Curl](https://github.com/curl/curl) is a command-line tool that lets you transfer data to and from a server using various protocols.
+
+> Curl is available in many distributions including Mac, Windows and Linux. Follow the install instructions in the [docs](https://curl.haxx.se/docs/install.html).
+
+<BlockSwitcher>
+<Block name="Mac and Linux">
+
+### GET method example
+
+```bash
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash
+curl -H "Content-Type: application/json" -d '{"name":"item-1"}' https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+<Block name="Windows">
+
+### GET method example
+
+```bash
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash
+curl -H "Content-Type: application/json" -d {\"name\":\"item-1\"} https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+</BlockSwitcher>
+
+> Important! Testing methods using production endpoints may result in changes to resources that cannot be undone.
+
+## Test the API with API Gateway console
+
+Let's test your new REST API using the route below with HTTP Method `GET` and path `/items?limit=10` which includes a `limit` query string parameter.
+
+```console
+GET /items?limit=10
+```
+
+> Important! Testing methods with the API Gateway console may result in changes to resources that cannot be undone.
+
+- Sign in to the API Gateway console at [https://console.aws.amazon.com/apigateway](https://console.aws.amazon.com/apigateway).
+- Choose the `myRestApi` REST API.
+- In the Resources pane, choose the method you want to test. Pick `ANY` right under `/items`.
```

Comment:
```suggestion
- Choose the `myRestApi` REST API
- In the Resources pane, choose the method you want to test. Select `ANY` right under `/items`
```

bullet points should not end with periods unless they are in sentence form
"pick" -> "select" (or "choose")